apisupport.ant/src/org/netbeans/modules/apisupport/project/universe/ModuleList.java

```diff
@@ -1263,6 +1263,18 @@ public static String findClusterLocation(File basedir, File nbroot, NbModuleType
                     clusterLocations.put(nbroot, clusterLocationsHere);
                 }
                 cluster = clusterLocationsHere.get(path);
```

Comment:
I'd like to get change in this file into version 9.0 - that way it should be possible to edit the new source structure in Apache NetBeans 9.0.